Add section on obsoleted stats.

alvestrand opened the pull request:
Closes #174

vr000m commented on webrtc-stats.html:
```diff
@@ -2096,6 +2117,36 @@
+        Obsolete stats
+      </h2>
+      <p>
+        This section's content is an example.
```
We have a concrete example, rtt got deprecated for roundTripTime, handle that as a separate issue?

alvestrand commented:
Harald will switch example in the "obsoleted" section to use rtt (which was renamed and moved).
Moved in #167 - renamed in #95 
We should say that we don't keep track of stuff that was never implemented - if we were pedantic, we would report things that were in drafts, but we don't want to do that.

alvestrand commented:
@vr000m still LGTM?

vr000m commented:
The #167 is related to the RTP RTT, we need another entry for RTCInboundRTPStreamStats.

In the case of #167, we changed the units from ms to seconds.
